scss/components/lfd/simple-list.scss

```
@@ -1019,6 +1019,17 @@
           background-color: map-get($configuration, simpleListFiltersOverlayBackground);
         }
 
+        .filter-holder {
+          .panel-default > .panel-heading {
+            color: map-get($configuration, simpleListOverlayFontColor);
+            background-color: map-get($configuration, simpleListFiltersOverlayBackground);
+            border-bottom: 1px solid rgba(map-get($configuration, simpleListOverlayFontColor), 0.2);
+          }
+          .panel-group .panel .panel-body {
```

```suggestion

          .panel-group .panel .panel-body {
```

Also, this doesn't seem to be using any value from the theme configuration. Can it not be added directly to the CSS file in the LFD component? Every property declaration in the theme SCSS file should use at least one value from a theme property.

---

@tonytlwu Yes, we can add this to the LFD component itself. Reason Why I added in the theme because all other layouts use the theme to add this behavior.
If I add this behavior to the LFD, should I clean other layouts SCSS files as well?

---

@YaroslavOvdii Yes, please. I'd like to keep the theme file as slim as possible because it bulks up the app CSS file very quickly when multiplied onto all 3 breakpoints.